docs/requirements.txt

```
@@ -1,6 +1,6 @@
-sphinx==4.1.0
-pydata_sphinx_theme==0.6.3
-myst-parser==0.15.1
+sphinx==7.1.0
+pydata_sphinx_theme==0.13.3
```

There is a known issue with RTD in the latest `pydata_sphinx_theme` so let's stick with the latest working one.
```suggestion
pydata_sphinx_theme==0.11.0
```

c.f. https://github.com/cupy/cupy/pull/7375

---

Fixed it!